.github/workflows/build.yml

```diff
@@ -41,6 +41,12 @@ jobs:
           ${{ runner.os }}-go-
     - name: Run test
       run: make test
+    - name: Upload Test Coverage
+      uses: codecov/codecov-action@v2
+      with:
+        token: ${{ secrets.CODECOV_TOKEN }}
+        flags: '${{ runner.os }},unit'
+        fail_ci_if_error: true
```

Comment:
Overriding the default such that this workflow fails anytime Codecov is unavailable seems unnecessary